invoices_app/filters.py

```diff
@@ -66,13 +67,22 @@ class InvoiceFilter(FilterSet):
         widget=CheckboxSelectMultiple(attrs={'class': 'list-unstyled'}),
     )
     invoice_date = DateFromToRangeFilter(widget=DateRangeWidget())
+    invoice_due_date = DateFromToRangeFilter(widget=DateRangeWidget())
+
     total_amount = RangeFilter(widget=NumericRangeWidget())
     taxpayer = ModelChoiceFilter(
         queryset=taxpayer_qs,
-        empty_label="All",
+        empty_label='All',
         widget=Select(attrs={'class': 'custom-select'}),
     )
 
+    taxpayer__country = CharFilter(
+        widget=Select(
+            choices=[('AR', 'Argentina'), ['BR', 'Brasil']],
```

Comment:
This is going to use the same constant defined in the Tax Payer model once it is defined.